ivadomed/loader/loader.py

```diff
@@ -307,22 +312,24 @@ def __init__(self, filename_pairs, slice_axis=2, cache=True, transform=None, sli
         """
         self.indexes = []
         self.filename_pairs = filename_pairs
-        self.transform = transform
+        if isinstance(transform, list):
+            self.prepro_transforms, self.transform = transform
+        else:
+            self.transform = transform
+            self.prepro_transforms = None
```

Comment:
That's for cases where we are not going through `load_dataset` where we have:
```
tranform_lst = [prepro_transforms if len(preprocessing_transforms) else None, transforms]
```
Correct? :-)

Reply:
yes exactly :)